### DISCUSSION TITLE
Names from Regions of the CIE Chromaticity Diagram

### DISCUSSION THREAD

**Discussion by cgrobb** [meodai/color-names#127] [General]
Hey folks,

My interest in color stems from my interest in displays, so I was interested to find today for the first time, this naming of major regions of the Chromaticity Diagram which is the way display color gamuts are typically compared.

http://hyperphysics.phy-astr.gsu.edu/hbase/vision/cie.html#c3

I imagine these region names are all in the master list here, but I do think it's interesting to give names to regions of colors, especially for discussion of varying display color gamuts.

**Comment by cgrobb** [comment]
![image](https://user-images.githubusercontent.com/805297/149557594-42588a17-35f4-47a8-973a-4df5fb72cf9c.png)

**Comment by meodai** [comment]
All those names are already in. It depends a lot on what distance function you would use on what name snaps to what color

**Comment by meodai** [comment]
@cgrobb I guess you had seen this: https://colornaming.net/
